src/LogstashFormatter.ts

```diff
@@ -0,0 +1,37 @@
+
+import { Entry } from 'aono';
+
+import { Formatter } from './Formatter';
+
+/**
+ * @author Maciej Chałapuk (maciej@chalapuk.pl)
+ */
+export class LogstashFormatter implements Formatter {
+  constructor(readonly consts : any = {}) {
+  }
+  format(entry : Entry) {
+    return '{ '+
```

Comment:
Shouldn't we use proper JSON serializer? Is this going to escape JSON properly?

Reply:
If by proper, you mean `JSON.stringify` then it is used everywhere where it's required. But maybe I should also use it on logger and message in order to escape utf characters.

Reply:
Yes, I mean `JSON.stringify`. What if e.g. `message` contains `"` character?